org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -623,6 +654,59 @@ private float IntersectQuad(Vector3 start, Vector3 end, float depth)
             }
         }
 
+        /// <summary>
+        /// Registers the duple Collider + XRBaseInteractor as triggering proximity.
+        /// </summary>
+        /// <param name="collider">Collider triggering proximity.</param>
+        /// <param name="xrBaseInteractor">Interactor triggering proximity.</param>
+        public void RegisterActiveColliderWithInteractor(Collider collider, XRBaseInteractor xrBaseInteractor)
+        {
+            if (collider != null && !activeCollidersWithInteractor.Contains((collider, xrBaseInteractor)))
```

Comment:
I forgot that behavior of the HashSets, thank you for catching that and letting me know.  Updated in latest commit.
I implemented:
```cs
if (collider != null &&
    activeCollidersWithInteractor.Add((collider, xrBaseInteractor)) &&
    activeCollidersWithInteractor.Count >= 1)
{
   SetEnabledDynamicComponents(true);
}
```
instead of:
`activeCollidersWithInteractor.Count == 1`

because there may be more than one type of combinations collider+xrBaseInteractor that are triggering proximity.
Updated in latest commit.